Src/VimCore/CoreTypes.fs

```diff
@@ -52,6 +52,27 @@ type JoinKind =
     | RemoveEmptySpaces
     | KeepEmptySpaces
 
+type NavigationKind =
+    | First = 1
+    | Last = 2
+    | Next = 3
+    | Previous = 4
+
+type ListKind =
+    | Error = 1

```

Comment:
Numbering should start at 0 here to follow the .NET Design Guidelines [https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum)

Reply:
Fixed.